_guide/_pages/security/aws.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Amazon Web Services (AWS) 
```

Comment:
Hmmmm, I'm wondering if presigned URLs are offered by other cloud storage companies beyond AWS? And if so, would it make sense to generalize this out from AWS to something broader? "Cloud security practices?"

Reply:
that's a good idea! i think other cloud providers use slightly different terminology but we can note that